cobalt/build/configs/linux-x64x11/args.gn

```
@@ -2,11 +2,21 @@ import("//cobalt/build/configs/chromium_linux-x64x11/args.gn")
 import("//cobalt/build/configs/common.gn")
 
 use_ozone = true
+ozone_auto_platforms = false
+ozone_platform = "starboard"
+angle_enable_gl = true
+
+# Required for tests
+if (!is_official_build) {
+  angle_use_x11 = true
```

Instead of having a conditional, you should do `angle_use_x11 = !is_official_build`. I'm a little worried this will be brittle and could silently fail at some point though, as it relies on `is_official_build` being set before `angle_use_x11` (right now it is in `gn.py`, but there's no guarantee). Less brittle but more verbose would be explicitly setting `angle_use_x11` to `true`/`false` in the `gn.py` script in the configuration dictionary values

---

Ok, just updated to set the values in the script dict instead